src/util/beautifier.ts

```diff
@@ -457,7 +457,7 @@ export default class Beautifier {
         m: 'isMaker',
         M: 'ignoreThis2',
         O: 'orderCreationTime',
-        Z: 'cumulativeQuoteAssetTransactedQty',
+        Z: 'cummulativeQuoteAssetTransactedQty',
```

Comment:
I'm tempted to leave the beautified ws properties as they were before this PR... Reasons:
- It's a breaking change (anyone that implemented this and is using the beautifier would need to update their code to use the renamed property)
- "cummulative" is incorrect spelling only seen in spot REST calls....and binance might fix this eventually.

You can keep both values in the list of float properties, but I think the payload from the ws beautifier should maintain the previously correctly spelt property "cumulativeQuoteAssetTransactedQty".